### PR TITLE
Task/176 create a scan code screen for wallet connect

### PR DESCRIPTION
**Pull request by TomasPetrik**

### CODE REVIEW

File: lib/view/screens/qr_code_scanner_screen.dart [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                  padding: const EdgeInsets.symmetric(
+                    horizontal: PaddingSize.padding16,
+                  ),
+                  child: QrAddressField(
+                    address: address,
+                  ),
+                )
+              ],
+            ),
+            const Spacer(),
```
Comment (jvrastil) [inline lib/view/screens/qr_code_scanner_screen.dart:109]: is it necessary here?

File: lib/view/themes/constants.dart [hunk 2]
```diff
@@ -9,6 +85,7 @@ class PaddingSize {
 
   static const double padding16 = 16;
   static const double padding32 = 32;
+  static const double padding48 = 48;
```
Comment (jvrastil) [inline lib/view/themes/constants.dart:88]: can you define al paddings(left, top,...) with the rest of new constants as well?

File: lib/view/widgets/qr_code_horizontal_line_animation.dart [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            child: BackdropFilter(
+              filter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
+              child: Container(
+                height: 1,
+                decoration: BoxDecoration(
+                  color: Theme.of(context).colorScheme.onPrimary,
+                  borderRadius: const BorderRadius.all(Radius.circular(20)),
+                  boxShadow: const [
+                    BoxShadow(
+                      color: Color.fromRGBO(55, 255, 198, 0.5),
```
Comment (jvrastil) [inline lib/view/widgets/qr_code_horizontal_line_animation.dart:54]: do we have some color constant for these two colors?

**Review by jvrastil — CHANGES_REQUESTED** [review]

File: pubspec.yaml [hunk 4]
```diff
@@ -40,6 +40,8 @@ dependencies:
   crypto_assets:
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply pubspec.yaml:40]: Okay.

File: lib/view/screens/qr_code_scanner_screen.dart (hunk 1 above)
Comment (TomasPetrik) [reply lib/view/screens/qr_code_scanner_screen.dart:109]: No necessary, you are right.

File: lib/view/themes/constants.dart (hunk 2 above)
Comment (TomasPetrik) [reply lib/view/themes/constants.dart:88]: Okey. Done.

File: lib/view/widgets/qr_code_horizontal_line_animation.dart (hunk 3 above)
Comment (TomasPetrik) [reply lib/view/widgets/qr_code_horizontal_line_animation.dart]: Good point. It was acutally mentolGreen color defined as inverseSurface in our colorScheme.

**Review by jvrastil — APPROVED** [review]

**Pull request merged**